adagrams/game.py

```diff
@@ -1,11 +1,100 @@
+import random 
+import copy
 def draw_letters():
-    pass
+    LETTER_POOL = {
+        'A': 9, 
+        'B': 2, 
+        'C': 2, 
+        'D': 4, 
+        'E': 12, 
+        'F': 2, 
+        'G': 3, 
+        'H': 2, 
+        'I': 9, 
+        'J': 1, 
+        'K': 1, 
+        'L': 4, 
+        'M': 2, 
+        'N': 6, 
+        'O': 8, 
+        'P': 2, 
+        'Q': 1, 
+        'R': 6, 
+        'S': 4, 
+        'T': 6, 
+        'U': 4, 
+        'V': 2, 
+        'W': 2, 
+        'X': 1, 
+        'Y': 2, 
+        'Z': 1
+    }
+#make list to return for "hand"
+#draw 10 random letters from pool
+#reflect letter distribution thats in dict
+    hand = []
+    for letter, amount in LETTER_POOL.items():
+        for num in range(amount):
+            hand.append(letter)
+            random.shuffle(hand)
+    return hand[:10]
```

Comment:
Love this approach! This is optimal and very handy, you're creating a large list full of all the values of each letter and then grabbing the first ten! 🎉